core/modules/css/DbInterfaceMySql.h

```diff
@@ -0,0 +1,68 @@
+// -*- LSST-C++ -*-
+/*
+ * LSST Data Management System
+ * Copyright 2019 LSST/AURA.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+#ifndef LSST_QSERV_CSS_DBINTERFACE_H
+#define LSST_QSERV_CSS_DBINTERFACE_H
+
+// System headers
+#include <mutex>
+#include <set>
+
+// Third-party headers
+
+// Qserv headers
+#include "mysql/MySqlConfig.h"
+#include "sql/SqlConnection.h"
+
+namespace lsst {
+namespace qserv {
+namespace css {
+
+
+/// This class is used to connect to tabular data in a mysql database.
+class DbInterfaceMySql {
+public:
+
+    /// @param mysqlConf: Configuration object for mysql connection
+    DbInterfaceMySql(mysql::MySqlConfig const& mysqlConf);
```

Comment:
This may need to be declared as `explicit` to avoid being used in the implicit conversions.